rad/src/types/mod.rs

```diff
@@ -5,7 +5,7 @@ use std::{
 };
 
 use cbor::value::Value as CborValue;
-use serde::Serialize;
+use log::warn;
```

Comment:
```suggestion
use log;
```
and later use log::warn!

Reply:
Fun fact: `use log;` doesn't do anything because dependencies are already imported. The code will work just as well without it.